server/src/concepts/user/model/User.js

```diff
@@ -0,0 +1,44 @@
+import mongoose from "mongoose";
+
+const userSchema = new mongoose.Schema({
+  _id: mongoose.Schema.Types.ObjectId,
```

Comment:
Hmm _id isn't in model by default? But Im not sure about that.

Reply:
It is :) I've just removed it from here